src/PowerSystems.jl

```diff
@@ -126,6 +126,7 @@ export ReserveUp
 export ReserveDown
 export StaticReserve
 export VariableReserve
+export ORDC
```

Comment:
Should choose a more descriptive name

Reply:
I know but `OperatingReserveDemandCurve` seem bit too long, so I would like some suggesting from you guys. May be just `ReserveDemandCurve` ?

Reply:
I like `ReserveDemandCurve`